Configuration/Geometry/README.md

```diff
@@ -85,25 +85,11 @@ The script also handles the common and forward elements of the geometry:
 * F6: same as F4 with modifications needed for BRM and forward shield
 
 Several detector combinations have been generated:
-* D49 = T15+C9+M4+I10+O4+F2
-* D50 = T15+C9+M4+I11+O4+F2
-* D60 = T15+C10+M4+I10+O4+F3
-* D64 = T22+C11+M4+I11+O5+F4
-* D65 = T23+C11+M4+I11+O5+F4
-* D66 = T21+C11+M8+I11+O5+F4
-* D67 = T21+C11+M9+I11+O5+F4
-* D68 = T21+C11+M6+I11+O5+F4
-* D69 = T21+C12+M6+I11+O5+F5
-* D70 = T21+C13+M7+I11+O6+F6
-* D71 = T21+C14+M7+I11+O7+F6
-* D72 = T21+C11+M6+I12+O5+F4
-* D74 = T21+C14+M9+I11+O7+F6
-* D75 = T21+C14+M7+I13+O7+F6
+* D49 = T15+C9+M4+I10+O4+F2 (HLT TDR baseline)
+* D60 = T15+C10+M4+I10+O4+F3 (With HFNose)
 * D76 = T21+C14+M9+I13+O7+F6
 * D77 = T24+C14+M9+I13+O7+F6
 * D78 = T22+C14+M9+I13+O7+F6
-* D79 = T23+C14+M9+I13+O7+F6
```

Comment:
@srimanob we had an afterthought with respect to what was discussed at https://github.com/cms-sw/cmssw/issues/33064#issuecomment-799341672 and possibly one geometry featuring Tracker T23 could still be useful because it's the only layout we have withh 3D pixels in the IT endcaps (we're debugging the template reco for 3D pixels at the moment, see more details [here](https://indico.cern.ch/event/992962/contributions/4293312/attachments/2218355/3756079/TkPh2Simulation20210330.pdf) ).
While it is possible to use a past pre-release for these studies it might still be useful to keep D79 around if it's not too inconvenient from the maintenance point of view.  In case not, we understand.
Thank you
cc: @emiglior 
@emiglior

Reply:
Done.

Reply:
Thanks!